pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/decoder/avro/TestAvroDecoder.java

```diff
@@ -127,6 +130,9 @@ public void testPrimitiveType() {
                 "enumField", VARCHAR, false, false, "enumField", null, null, PulsarColumnHandle.HandleKeyValueType.NONE);
         checkValue(decodedRow, enumFieldColumnHandle, message.enumField.toString());
 
+        PulsarColumnHandle decimalFieldColumnHandle = new PulsarColumnHandle(getPulsarConnectorId().toString(),
+                "decimalField", DecimalType.createDecimalType(4, 2), false, false, "decimalField", null, null, PulsarColumnHandle.HandleKeyValueType.NONE);
+        checkValue(decodedRow, decimalFieldColumnHandle, message.decimalField);
```

Comment:
Since there are two implementation of DecimalType: `ShortDecimalType` and `LongDecimalType`. And they are mapping two different type: `Long` and `Slice`.  But it looks like only the Long type was tested here. It's better to add a test case for the Slice.

Reply:
Ok, i will add it.